docs/source/notebooks/probabilistic_matrix_factorization.ipynb

```diff
@@ -0,0 +1,1489 @@
+{
```

Comment:
<p>i'd update this to use just <code>pm.sample(draws=500, tune=500)</code> (or similar!)</p><p> </p><p>This should automatically use 1 chain per physical core and run diagnostics.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>